proc_CPMG.py

```diff
@@ -23,10 +23,8 @@
     fl.next('centered echo')
     fl.image(s)
     #{{{select echo decay fit function
-    s = s['t2':(0,None)]
-    s['t2',0] *= 0.5
     s.ft('t2')
-    data = s['t2':(0,None)].sum('t2')
+    data = s['t2':f_slice].sum('t2')
```

Comment:
the slice from 0 Hz to the highest frequency (what was here before) doesn't make any sense.  I assume you want a frequency bandwidth slice here, so I specify one above.